deny.toml

```diff
@@ -161,9 +160,6 @@ skip = [
     # actix-http hasn't upgraded iminz_oxide/flate2 yet.
     { name = "miniz_oxide", version = "=0.5.1" },
 
-    # webrtc-util depends on old version of nix depends on old version of memoffset
```

Comment:
Admittedly if we’re not keen on maintaining these annotations/comments, then we’d be better off with none at all. It isn’t particularly hard to figure out where a dependency comes from later on with `cargo tree` or some other such tool.

Reply:
In particular now that `memoffset` has been moved to the block with "wasmer/wasmtime", the attribution isn’t exactly correct anymore.

Reply:
I totally agree, but didn’t really want to remove them all without discussion, so I only removed the ones I know we should be mostly responsible for :)